drivers/bluetooth/hci/vendor/infineon/Kconfig

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) 2022 Cypress Semiconductor Corporation (an Infineon company) or
+# an affiliate of Cypress Semiconductor Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Change size of command lengths. It for vendor commands related to
+# firmware downloading.
+config BT_BUF_CMD_TX_SIZE
```

Comment:
Could you remove the `vendor/infineon` directory? Here's my proposal:
driver/bluetooth/hci
  CMakeLists.txt
  Kconfig
  Kconfig.infineon
  cyw43xxx.c

The `Kconfig.vendor` or `Kconfig.model` convention is used in other drivers (e.g. look in `drivers/led`), so I think we should also apply it here.

Reply:
hi @jori-nordic,
Yes i can update structure.

cyw43xx is wifi/bt connectivity device, this MR covers BT driver, also i will push WIFi part. If look at drivers wifi folder, there are grouping by vendors. So i applied similar here.

Reply:
Alright I see that now. You can keep the vendor/infineon structure then, but then could you add another commit that moves the other vendor-specific drivers into e.g. vendor/st, vendor/espressif etc..?

Reply:
@tejlmand @Thalley @carlescufi any of you have a preference?

Reply:
Hi @tejlmand @Thalley @carlescufi Do you have any comments here? if no, I am ok to leave as is...

Reply:
Let's leave it as-is :+1: